packages/phone-number-privacy/combiner/src/bls/bls-cryptography-client.ts

```diff
@@ -57,9 +57,15 @@ export class BLSCryptographyClient {
     }
     // Optimistically attempt to combine unverified signatures
     // If combination fails, iterate through each signature and remove invalid ones
-    // We do this since signature verification incurs higher latencies
+    // We do this since partial signature verification incurs higher latencies
     try {
       const result = threshold_bls.combine(threshold, this.allSignatures)
+      // TODO(Alec): confirm this works if message is still blinded and document to avoid confusion
+      threshold_bls.verify(
```

Comment:
We need to use `verifyBlindSignature` because the `verify` function expects the input to be the original (unblinded) message and will hash the input, which will cause verification to fail.

https://github.com/celo-org/blind-threshold-bls-wasm/blob/master/src/blind_threshold_bls.d.ts#L133-L148

Reply:
The doc string does say:
> Verifies the signature **after it has been unblinded** without hashing.

But it appear this may just be incorrect documentation. I'll need to check that. @kobigurk would also know.

Reply:
Ya, I'm _fairly_ certain this is simply incorrect documentation. The implementation looks like it verifies a blinded signature over a blinded message, as the name suggests:
* `blind_verify` https://github.com/celo-org/celo-threshold-bls-rs/blob/master/crates/threshold-bls/src/sig/blind.rs#L85-L93
* Compared to `verify` https://github.com/celo-org/celo-threshold-bls-rs/blob/master/crates/threshold-bls/src/sig/bls.rs#L57-L72